### DISCUSSION TITLE
Can't login via extension anymore

### DISCUSSION THREAD

**Discussion by orangebutblue** [dani-garcia/vaultwarden#3023] [Q&A] [answered]
I can't log into my vaultwarden server anymore via the extension (chrome/brave) on a new PC. It still works fine on all of my other devices, however I haven't fully logged out of any of them.
Web login works fine, but via the extension, I get an "incorrect username/password" error.

```
[2022-12-21 09:20:48.078][vaultwarden::api::identity][ERROR] Username or password is incorrect. Try again. IP: 192.168.222.1. Username: mymail@server.com.,
[2022-12-21 09:20:48.078][response][INFO] POST /identity/connect/token (login) => 400 Bad Request,
```

I am running 1.23.0 - I realize that this version is a few months old. However, it's the latest armv6 version as of now.  Is the testing-arm32v6 tag safe to test?

**Comment by BlackDex** [comment, marked as answer]
That is definitely not the latest version, not even for the old `bitwrdenrs` docker hub repo.
So, double check where you get the image from and use the right repo, which is https://hub.docker.com/r/vaultwarden/server

Second, the extension is probably updated to the latest version of Bitwarden Browser Extensions, which needs the `testing` tagged images for now. We are preparing to release a new version soon (no definitive date/time determined), until that time, use the testing tagged images.

**Comment by orangebutblue** [comment]
I'm sorry, I don't know what happened. it seems like the pulling process (for the newest image) would get killed (probably due to insufficient storage) and I kept thinking that it just pulled the newest image correctly. That's why I was still on 1.23

I just pulled it correctly and I'm on 1.26 and the extension connection now works correctly again. Thank you
